### PR TITLE
Bugfix for fixed distribution

### PR DESCRIPTION
**Pull request by mpenkows**

### CODE REVIEW

File: include/ze_info/client.hpp [hunk 1]
```diff
@@ -47,7 +47,7 @@ class client
 
 				// read the elements in the file into a vector  
 				int n = 0;
-				while (inputFile >> value) {
+				while (inputFile >> value && n < queries) {
```
Comment (tponieck) [inline include/ze_info/client.hpp]: Why do we need this additional condition? for a case when someone want to use dist from run with different number of queries?

Comment (mpenkows) [reply include/ze_info/client.hpp]: right. currently you can get error connected to vector length

Comment (tponieck) [reply include/ze_info/client.hpp]: hmm, IMHO it should be done in that way:
if size doesn't match. delete old, create new file - prompt that size did match and new one has been created

Comment (mpenkows) [reply include/ze_info/client.hpp]: I've just pushed the solution discussed offline

**Pull request merged**